<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partSettingsConsole</class>
 <widget class="QWidget" name="partSettingsConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chkConsoleShowProgramCommands">
     <property name="text">
      <string>Show G-code program commands (may impact performance!!)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkConsoleShowUICommands">
     <property name="text">
      <string>Show UI commands</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkConsoleShowSystemCommands">
     <property name="text">
      <string>Show system commands</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkConsoleAutocompletion">
     <property name="text">
      <string>Command auto-completion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkConsoleDarkMode">
     <property name="text">
      <string>Dark mode (black background, white letters)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
